.github/workflows/dapr-bot.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Comment analyzer
-        uses: actions/github-script@v1
+        uses: actions/github-script@v6
```

Comment:
@berndverst line 30 has a breaking change in v4 to v5 of GitHub actions ... 
https://github.com/actions/github-script#breaking-changes-in-v5